docs/conf.py

```diff
@@ -0,0 +1,59 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import subprocess
+import sys
+from pathlib import Path
+
+# Assuming conf.py is located in /Users/sergeypetrakov/Documents/Documents/Skoltech_PhD/rtd_to_main/lm-polygraph/src/lm_polygraph/docs
```

Comment:
This should be portable

Reply:
this comment was just about the fact that conf.py must be in docs directory